README.md

```diff
@@ -77,3 +77,18 @@ To run these integration tests locally:
 
 > **Note:** you can specify `CODER_IMAGE` if the Coder image you wish to test is hosted somewhere other than `ghcr.io/coder/coder`.
 > For example, `CODER_IMAGE=example.com/repo/coder CODER_VERSION=foobar make test-integration`.
+
+### How to create a new release
+> **Warning:** Before creating a new release, make sure you have pulled the latest commit from the main branch.
```

Comment:
```suggestion
> **Warning:** Before creating a new release, make sure you have pulled the latest commit from the main branch i.e. `git pull origin main`
```

Just in case.